core/tzdbOnFilesystem/src/internal/TzdbOnFilesystem.kt

```diff
@@ -47,7 +47,9 @@ internal fun tzdbPaths(defaultTzdbPath: Path?) = sequence {
     currentSystemTimeZonePath?.splitTimeZonePath()?.first?.let { yield(it) }
 }
 
-internal val currentSystemTimeZonePath get() = chaseSymlinks("/etc/localtime")
+internal var root: String = "/"
```

Comment:
This variable should either have a descriptive name explaining the limited scope it's applied to (that is, something like `systemTimezoneSearchRoot`?) or be applied consistently to all filesystem access (for example, to the list of locations to search for a timezone database in this file).